### DISCUSSION TITLE
How do I connect to a pscale db with Sequel Pro?

### DISCUSSION THREAD

**Discussion by corysimmons** [planetscale/discussion#205] [Q&A] [answered]
I've got my credentials all set up, but I think I need to connect with SSL.

The docs I read just say something to the extent of _"just add a cert.pem, et voila"_ but I'm not sure how to do that. Are there any docs for this that I'm overlooking?

**Comment by jgreet** [comment, marked as answer]
@corysimmons the best option is probably to move to the updated fork Sequel Ace, which has much better SSL support:
https://github.com/Sequel-Ace/Sequel-Ace

See also this [previous discussion](https://github.com/planetscale/discussion/discussions/90#discussioncomment-1234888).

**Reply by corysimmons** [reply]
Thank you I'll give it a shot.
